documentation/onboarding.md

```diff
@@ -0,0 +1,147 @@
+
+The installation process begins. Allow a few minutes for the installation to complete.
+
+![The install status bar, showing the progression bar.](./images/onboarding/installing.png)
+
+Agenda is successfully installed on the system and a shortcut icon is created on the desktop.
+
+![The desktop shortcut on the Windows desktop.](./images/onboarding/desktop-shortcut.png)
+
+You can now proceed to the next step, [creating an account](#create)
+
+### MacOS install {#mac-install}
+
+The Agenda application is compatible with MacOS and can be installed on the MacOS Catalina and newer operating systems.
+
+Using the web browser of your choice, navigate to [the official Agenda download site](https://www.example.agenda.ca/download-app) to download the Agenda desktop client on MacOS.Select **[!UICONTROL Download]** under the corresponding section for the MacOS operating system.
+
+![The download site, with the MacOS download button highlighted.](./images/onboarding/download-center-mac.png)
+
+The file is downloaded and saved to your machine.
+
+Navigate to the **Downloads** folder (or your alternate downloads folder if you changed the default) and select the installer.
+
+![The downloaded installer in the download location.](./images/onboarding/launch-installer-mac.png)
+
+The installer dialog opens. Follow the installer's instructions until you arrive at the Installation step. Enter your administrator details (Username and Password) for your device and select **[!UICONTROL Install Software]**.
+
+![The installer credentials dialog, with the Install Software button highlighted.](./images/onboarding/installer-credentials.png)
+
+Allow a few minutes for the installation to complete. A dialog appears confirming the installation was successful. Select **[!UICONTROL Close]**.
+
+![The installer dialog on the Summary tab confirming installation was successful, with the Close button highlighted.](./images/onboarding/installation-successful.png)
+
+The Agenda app is now available in Launchpad.
+
+![The Agenda app icon in the Launchpad](./images/onboarding/launchpad.png)
+
+## Create an account {#create}
+
+An Agenda account gives you access to all the app's features. With an Agenda account you can do the following:
+
+* Manage meetings and huddles
+* Manage teams
+* Integrate external calendars
+* Manage multiple calendars in one space
+
+To create a new account, navigate to the desktop application and select to open. On the sign-in screen that appears, select **[!UICONTROL Create an account]**.
+
+![The Agenda desktop app Sign-In screen, with the Create and account button highlighted.](./images/onboarding/create-an-account.png)
+
+Enter your name and email address, then choose a password and confirm that same password before selecting **[!UICONTROL Create account]**.
+
+![The Create an account screen, with required fields and the Create a account button highlighted.](./images/onboarding/complete-create-account.png)
```

Comment:
Can we update this screenshot to show all the form fields filled out instead of blank?